pandas/tests/arithmetic/test_numeric.py

```diff
@@ -0,0 +1,534 @@
+
+    # TODO: de-duplicate with test_numeric_arr_mul_tdscalar
+    def test_ops_series(self):
+        # regression test for G#H8813
+        td = Timedelta('1 day')
+        other = pd.Series([1, 2])
+        expected = pd.Series(pd.to_timedelta(['1 day', '2 days']))
+        tm.assert_series_equal(expected, td * other)
+        tm.assert_series_equal(expected, other * td)
+
+    @pytest.mark.parametrize('box', [
+        pd.Index,
+        Series,
+        pytest.param(pd.DataFrame,
+                     marks=pytest.mark.xfail(reason="block.eval incorrect",
+                                             strict=True))
+    ])
+    @pytest.mark.parametrize('index', [
+        pd.Int64Index(range(1, 11)),
+        pd.UInt64Index(range(1, 11)),
+        pd.Float64Index(range(1, 11)),
+        pd.RangeIndex(1, 11)],
+        ids=lambda x: type(x).__name__)
+    @pytest.mark.parametrize('scalar_td', [
+        Timedelta(days=1),
+        Timedelta(days=1).to_timedelta64(),
+        Timedelta(days=1).to_pytimedelta()],
+        ids=lambda x: type(x).__name__)
+    def test_numeric_arr_mul_tdscalar(self, scalar_td, index, box):
+        # GH#19333
+
+        if (box is Series and
+                type(scalar_td) is timedelta and index.dtype == 'f8'):
+            raise pytest.xfail(reason="Cannot multiply timedelta by float")
+
+        expected = pd.timedelta_range('1 days', '10 days')
+
+        index = tm.box_expected(index, box)
+        expected = tm.box_expected(expected, box)
+
+        result = index * scalar_td
+        tm.assert_equal(result, expected)
+
+        commute = scalar_td * index
+        tm.assert_equal(commute, expected)
+
+    @pytest.mark.parametrize('index', [
+        pd.Int64Index(range(1, 3)),
+        pd.UInt64Index(range(1, 3)),
+        pd.Float64Index(range(1, 3)),
+        pd.RangeIndex(1, 3)],
+        ids=lambda x: type(x).__name__)
+    @pytest.mark.parametrize('scalar_td', [
+        Timedelta(days=1),
+        Timedelta(days=1).to_timedelta64(),
+        Timedelta(days=1).to_pytimedelta()],
+        ids=lambda x: type(x).__name__)
+    def test_numeric_arr_rdiv_tdscalar(self, scalar_td, index, box):
+
+        if box is Series and type(scalar_td) is timedelta:
+            raise pytest.xfail(reason="TODO: Figure out why this case fails")
+        if box is pd.DataFrame and isinstance(scalar_td, timedelta):
+            raise pytest.xfail(reason="TODO: Figure out why this case fails")
+
+        expected = TimedeltaIndex(['1 Day', '12 Hours'])
+
+        index = tm.box_expected(index, box)
+        expected = tm.box_expected(expected, box)
+
+        result = scalar_td / index
+        tm.assert_equal(result, expected)
+
+        with pytest.raises(TypeError):
+            index / scalar_td
+
+
+# ------------------------------------------------------------------
+# Arithmetic
+
+class TestDivisionByZero(object):
+
+    def test_div_zero(self, zero, idx):
```

Comment:
are these specific to Index tests? or are they moving so that you can consolidate later?

Reply:
> or are they moving so that you can consolidate later?

Yes, very nearly everything in this PR is moved with minimal changes so they can be de-duplicated and parametrized/fixturized in an upcoming pass.